frontend/pages/index.tsx

```diff
@@ -48,228 +47,265 @@ type HomeProps = {
-          <div
-            className="flex gap-10 w-full max-w-screen-xl flex-wrap mt-6 p-3 md:p-10 items-center opacity-30">
-            <LogoEscience className="max-w-[160px]"/>
-            <LogoHelmholtz className="max-w-[130px]"/>
-            <LogoUMC className="max-w-[200px]"/>
-            <LogoUU className="max-w-[220px]"/>
-            <LogoLeiden className="max-w-[220px]"/>
+        {/* Software spotlights */}
+        <div className="conainer mx-auto p-6 md:p-10 max-w-screen-xl text-secondary">
+          <h2 className='text-5xl'>Software Spotlights</h2>
+          <div className='text-2xl mt-2'>Browse the latest outstanding software products in Helmholtz</div>
+          <div className="w-full">
+            <Spotlights spotlights={SPOTLIGHTS} />
           </div>
         </div>
 
-        {/* Footer */}
-        <div className="mt-20">
-          <AppFooter/>
-        </div>
+        {/* Software meta repository */}
+        {/* <div className="conainer mx-auto p-12 max-w-screen-xl text-secondary bg-[#ecfbfd]">
+          <h2 className='text-5xl'>Software Meta Repository</h2>
+          <div className="text-xl my-4">Browse Software by Research Topic</div>
+          <p className="text-secondary text-xl my-6">
+            The Research Software Directory is originally developed at the netherlands eScience center. < br/>
+            In a joint collaboration, Helmholtz offers this portal that allows Research Software Engineers to promote their software.
+          </p>
+          <div className="flex justify-end mt-12">
+            <Link
+              href="/software"
+              passHref
+            >
+              <a>
+                <div className="w-[250px] bg-[#05e5ba] hover:bg-primary text-secondary hover:text-white text-center font-medium text-2xl py-4 px-6 rounded-sm">
+                  Discover software
+                </div>
+              </a>
+            </Link>
+          </div>
+        </div> */}
+
+        {/* Roadmap */}
+        {/* <div className="bg-white">
+          <div className="conainer mx-auto py-6 px-4 max-w-screen-xl text-secondary">
+            <h1 className="pb-4">Roadmap</h1>
+            <p className="prose">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Proin libero nunc consequat interdum. Pretium vulputate sapien nec sagittis aliquam malesuada bibendum arcu. Id volutpat lacus laoreet non curabitur gravida arcu. Amet mauris commodo quis imperdiet massa tincidunt nunc. Varius sit amet mattis vulputate. Suscipit adipiscing bibendum est ultricies integer. Hendrerit gravida rutrum quisque non tellus. Eget felis eget nunc lobortis mattis aliquam. Integer enim neque volutpat ac tincidunt vitae. Condimentum id venenatis a condimentum vitae sapien pellentesque habitant.</p>
+          </div>
+        </div> */}
+
+        <AppFooter/>
```

Comment:
#2 mentions *Participating Centres* and *Partner Organisations* as requirements. They are still missing.

Reply:
Fair point, I am not sure how to implement the organisations and centres on the frontpage. They are shown on the "Organisations" page. Let's not close #2 with this PR.